R/get_cells.R

```diff
@@ -3,14 +3,18 @@ get_cells <- function(ss,
                       sheet = NULL,
                       range = NULL,
                       col_names_in_sheet = TRUE,
-                      skip = 0, n_max = Inf) {
+                      skip = 0, n_max = Inf,
+                      detail_level = c("default", "full"),
+                      discard_empty = TRUE) {
```

Comment:
This controls whether we discard empty cells, i.e. cells that have no value. Usually will be `TRUE`, unless you really want to look at formats.